packages/backend/src/graphql/resolvers/asset.test.ts

```diff
@@ -1,4 +1,4 @@
-import { gql } from 'apollo-server-koa'
+import { gql } from '@apollo/client'
```

Comment:
`@apollo/server` does not export the `gql` tag. 

Importing this from `@apollo/client` since it's only used for tests, so we don't have to install another dependency (`graphl-tag`).